vagrant_box_defaults.rb

```diff
@@ -8,3 +8,5 @@
 $NETNEXT_SERVER_VERSION= "53"
 @v419_SERVER_BOX= "cilium/ubuntu-4-19"
 @v419_SERVER_VERSION= "12"
+@v49_SERVER_BOX="cilium/ubuntu"
```

Comment:
Isn't this part of https://github.com/cilium/cilium/pull/11061?

Reply:
yeah, will probably need to rebase after it's merged